code/__HELPERS/unsorted.dm

```diff
@@ -58,6 +58,9 @@
 	else if(dx<0)
 		.+=360
 
+/proc/inverse_dir(dir)
+	return turn(dir, 180)
```

Comment:
Why not macro this instead of making it a proc?

Reply:
I think a macro for this already exists anyway.

Reply:
What's the macro then? I would prefer to remove this (unused) proc than make a duplicate definition.

Reply:
Adding it to these would be positive:
```#define ISDIAGONALDIR(d) (d&(d-1))
#define NSCOMPONENT(d)   (d&(NORTH|SOUTH))
#define EWCOMPONENT(d)   (d&(EAST|WEST))
#define NSDIRFLIP(d)     (d^(NORTH|SOUTH))
#define EWDIRFLIP(d)     (d^(EAST|WEST))```